src/sysmem/exports.exp

```diff
@@ -152,6 +152,7 @@ PSP_EXPORT_FUNC_NID(sceKernelSetUsersystemLibWork, 0xA6848DF8)
 PSP_EXPORT_FUNC_NID(sceKernelTotalMemSize, 0xACBD88CA)
 # sceKernelFreePartitionMemory
 PSP_EXPORT_FUNC_NID(sceKernelFreePartitionMemoryForUser, 0xB6D61D02)
+# sceKernelSafetyCheck0 according to PS3 NIDs, but doesn't match SHA1
```

Comment:
Maybe because it was only added after salting began?

Reply:
IIRC other functions appeared in that userland library afterwards but were not salted, while this one's NID didn't even change with firmware versions. So I'm not sure what to think, but I guess it's possible they applied the salt by hand and so the userland functions can have or not have the salt, changing or not with firmware versions...